website/src/components/HardForkTime/index.tsx

```diff
@@ -8,10 +8,10 @@ export interface HardForkTimeProps {
 }
 
 export default function HardForkTime({
-  mainnetDate = "TBD",
-  mainnetEpoch = "TBD",
-  testnetDate = "TBD",
-  testnetEpoch = "TBD",
+  mainnetDate = "July 1, 2025",
```

Comment:
I don’t think we need this. the default value in the HardForkTime function should remain 'TBD'